tracks/bash/exercises/grains/mentoring.md

```diff
@@ -0,0 +1,92 @@
+# Grains
+Hello World -> **Grains** -> Armstrong Numbers -> Hamming -> ...
```

Comment:
Could you remove this line please? It's something we will automatically inject into the website at a later way and I don't want the technical debt of it being included in text documents.